app/views/ConfirmSubmission.scala.html

```diff
@@ -112,25 +112,32 @@ <h2 class="govuk-heading-m">@messages("confirm_submission.declarationHeading")</
   if(nIProtocolEnabled) ".NIProtocol" else ""
 }
 
-@mainTemplate(pageTitle = messages("confirm_submission.title"), appConfig = appConfig, user = Some(user)) {
+@title = @{
+  messages("confirm_submission.heading") +
+  displayDateRange(viewModel.returnDetail.start, viewModel.returnDetail.end, useShortDayFormat = true) + " " +
+  messages("confirm_submission.returnDueDate", displayDate(viewModel.returnDetail.due))
+}
+
+@mainTemplate(pageTitle = title, appConfig = appConfig, user = Some(user)) {
 
   @govukBackLink(BackLink(
     content = Text(messages("common.back")),
     href = controllers.routes.SubmitFormController.show(viewModel.periodKey).url
   ))
 
-    <h1 class="govuk-heading-xl">
+  <h1 class="govuk-heading-xl">
     <span class="govuk-caption-xl">@messages("confirm_submission.heading")</span>
-        @displayDateRange(viewModel.returnDetail.start, viewModel.returnDetail.end, useShortDayFormat = true)
+    @displayDateRange(viewModel.returnDetail.start, viewModel.returnDetail.end, useShortDayFormat = true)
     <span class="govuk-caption-m">@messages("confirm_submission.returnDueDate", displayDate(viewModel.returnDetail.due))</span>
-    </h1>
+  </h1>
 
-    @validationWarning
+  @validationWarning
 
-    <h2 class="govuk-heading-l">@viewModel.userName</h2>
-    <h3 class="govuk-heading-s">@messages("confirm_submission.vatDetails")</h3>
+  <h2 class="govuk-heading-l">@viewModel.userName</h2>
+  <h3 class="govuk-heading-s">@messages("confirm_submission.vatDetails")</h3>
+  <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
```

Comment:
Can you remove `govuk-section-break--l` class to get this line to match the vertical spacing of the rows beneath it